examples/add_public_raster_dataset.ipynb

```diff
@@ -0,0 +1,304 @@
+    "          \"4\": \"Deciduous Broadleleaf Forests\",\n",
+    "          \"5\": \"Mixed Forests\",\n",
+    "          \"6\": \"Closed Shrublands\",\n",
+    "          \"7\": \"Open Shrublands\",\n",
+    "          \"8\": \"Woody Savannas\",\n",
+    "          \"9\": \"Savannas\",\n",
+    "          \"10\": \"Grasslands\",\n",
+    "          \"11\": \"Permanent Wtlands\",\n",
+    "          \"12\": \"Croplands\",\n",
+    "          \"13\": \"Urban and Built-Up\",\n",
+    "          \"14\": \"Cropland-Natural Vegetation Mosaics\",\n",
+    "          \"15\": \"Snow and Ice\",\n",
+    "          \"16\": \"Barren or Sparsely Vegetated\"\n",
+    "        }\n",
+    "      },\n",
+    "      \"noDataValue\": 255.0,\n",
+    "      \"time\": {\n",
+    "        \"start\": \"-262144-01-01T00:00:00+00:00\",\n",
+    "        \"end\": \"+262143-12-31T23:59:59.999+00:00\"\n",
+    "      },\n",
+    "      \"bbox\": {\n",
+    "        \"upperLeftCoordinate\": [-180.0, 90.0],\n",
+    "        \"lowerRightCoordinate\": [180.0, -90.0]\n",
+    "      },\n",
+    "      \"resolution\": {\n",
+    "        \"x\": 0.1,\n",
+    "        \"y\": 0.1\n",
+    "      }\n",
+    "    }\n",
+    "\n",
+    "result_descriptor = ge.RasterResultDescriptor(result_descriptor_dict)\n",
+    "\n",
+    "meta_data = ge.GdalMetaDataStatic(None, gdal_params, result_descriptor)\n",
+    "\n",
+    "meta_data.to_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add raster dataset the the first volume"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'volume': 'f6aa9f3d-a211-43e8-9b91-b23ab9632791', 'definition': {'properties': {'name': 'landcover', 'description': '', 'sourceOperator': 'GdalSource'}, 'metaData': {'type': 'GdalStatic', 'time': None, 'params': {'filePath': 'raster/landcover.tif', 'rasterbandChannel': 1, 'geoTransform': {'originCoordinate': {'x': -180.0, 'y': 90.0}, 'xPixelSize': 0.1, 'yPixelSize': -0.1}, 'width': 3600, 'height': 1800, 'fileNotFoundHandling': 'NoData', 'noDataValue': None, 'propertiesMapping': None, 'gdalOpenOptions': None, 'gdalConfigOptions': None, 'allowAlphabandAsMask': True}, 'resultDescriptor': {'type': 'raster', 'dataType': 'U8', 'measurement': {'type': 'classification', 'measurement': 'Land Cover', 'classes': {0: 'Water Bodies', 1: 'Evergreen Needleleaf Forests', 2: 'Evergreen Broadleaf Forests', 3: 'Deciduous Needleleaf Forests', 4: 'Deciduous Broadleleaf Forests', 5: 'Mixed Forests', 6: 'Closed Shrublands', 7: 'Open Shrublands', 8: 'Woody Savannas', 9: 'Savannas', 10: 'Grasslands', 11: 'Permanent Wtlands', 12: 'Croplands', 13: 'Urban and Built-Up', 14: 'Cropland-Natural Vegetation Mosaics', 15: 'Snow and Ice', 16: 'Barren or Sparsely Vegetated'}}, 'spatialReference': 'EPSG:4326'}}}}\n"
```

Comment:
Could we format the representation of the dataset id?
It looks super ugly to have this unformatted JSON in the Jupyter notebook sweat_smile :

Or is this just a lost println?

Reply:
it was just a lost print